experiments/test1_read_with_a.py

```
@@ -0,0 +1,63 @@
+    # Ajouter des métadonnées manuellement
+    print("\n1. Ajout de métadonnées initiales...")
+    cmd1 = [
+        "exiftool", "-overwrite_original",
+        "-XMP-iptcExt:PersonInImage=Anthony",
+        "-XMP-iptcExt:PersonInImage+=Bernard", 
+        "-XMP-dc:Subject=Anthony",
+        "-XMP-dc:Subject+=Bernard",
+        "-XMP-dc:Subject+=Album: Vacances",
+        str(test_file)
+    ]
+    subprocess.run(cmd1, capture_output=True)
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)